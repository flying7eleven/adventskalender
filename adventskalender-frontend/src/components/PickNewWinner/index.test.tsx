import React from 'react';
import { create } from 'react-test-renderer';
import { PickNewWinner } from './index';

describe('PickNewWinner component', () => {
    test('Matches the snapshot', () => {
        const component = create(<PickNewWinner isLoadingNewWinner={false} label={'Some text'} onRequestWinner={() => {}} />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
