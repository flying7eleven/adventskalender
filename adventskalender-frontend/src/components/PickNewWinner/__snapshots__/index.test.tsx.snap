// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PickNewWinner component Matches the snapshot 1`] = `
<button
  className="MuiLoadingButton-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root css-1twrztz-MuiButtonBase-root-MuiButton-root-MuiLoadingButton-root"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  Pick a new winner
</button>
`;
